### DISCUSSION TITLE
What's the state of the project?

### DISCUSSION THREAD

**Discussion by Slion** [Snoothy/UCR#278] [Development]
I'm considering contributing a GearVR Controller input extension at some point.
However I'm a bit concerned by the state of the project.
No commit on master for a couple of years now…
Which branch should I use as a base for new contributions?
Project looks awesome though.

**Comment by Slion** [comment]
Looks like the develop branch has quite a few commits past master:
https://github.com/Snoothy/UCR/commits/develop
Last commit there was like a year ago.

**Reply by Tactile-Taco** [reply]
I had interest in the project about a year ago but they were having issues with Appveyor and merging the dev branch. AutoHotInterception still has active development but this project for the most part seems dead.

**Reply by evilC** [reply]
Yep, accurate. I do not have control of the repo (I only have control of the IOWrapper repo, which is the back end) and I am unable to do merges because of broken CI
I have tried to get it sorted, but had no luck, so unfortunately it seems the project is dead :(

**Reply by Slion** [reply]
@Luke-Hill-Tactical-Taco @evilC Thanks for the heads up.

So it looks like @Snoothy did not work on the project for about a year, that hardly makes it dead right? I mean anyone including myself could pick it up and dust it off right?

There is a fork there https://github.com/HidWizards/UCR but it looks like that belongs to @Snoothy too.

**Reply by evilC** [reply]
Yeah - I did have a play around with trying to work out how to clone the repo to the Hidwizards org (Where IOWrapper lives) with all branches and all PRs, but I could not work out how to do it. Is it even possible?

**Reply by Slion** [reply]
You should get all the branches at the time of the fork. You won't get the PRs though and you need to ask GitHub support to disconnect it from the fork to be able to manage issues in your fork. There must be a way to pick up the PRs maybe do that before disconnecting it.

So you do own HidWizards? In the people section I could only see @Snoothy .

**Reply by Slion** [reply]
@evilC Looks like [HidWizards](https://github.com/HidWizards) is not the right place to maintain UCR in the future as it looks like it belongs to @Snoothy too and he is still MIA.

**Comment by darvon123** [comment]
So, this is an abandoned project now?
over 3 years of No updates is painting a grim picture.
I don't even think this @Snoothy user is even active now.
man, all I wanted to do was Test out NDOF mode in blender without having to buying some $200 dollar accessories.
There got to be another project I could use.
it really does feel like I am grasping for straws here, but I can't give up I'm into deep.
Maybe I should stick to this App and just hope it just work out.
